src/test/java/de/mediathekview/mserver/crawler/basic/M3U8ParserTest.java

```diff
@@ -13,38 +15,76 @@
   public void parseTestSrf() {
 
     M3U8Dto[] expected = new M3U8Dto[]{
-      createDto("https://hdvodsrforigin-f.akamaihd.net/i/vod/1gegen100/2010/05/1gegen100_20100517_200706_web_h264_16zu9_,lq1,mq1,hq1,.mp4.csmil/index_0_av.m3u8",
-      new String[]{M3U8Constants.M3U8_PROGRAM_ID, M3U8Constants.M3U8_BANDWIDTH, M3U8Constants.M3U8_CODECS, M3U8Constants.M3U8_CLOSED_CAPTIONS},
-      new String[]{"1", "118000", "\"avc1.66.30, mp4a.40.2\"", "NONE"}),
-      createDto("https://hdvodsrforigin-f.akamaihd.net/i/vod/1gegen100/2010/05/1gegen100_20100517_200706_web_h264_16zu9_,lq1,mq1,hq1,.mp4.csmil/index_1_av.m3u8",
-      new String[]{M3U8Constants.M3U8_PROGRAM_ID, M3U8Constants.M3U8_BANDWIDTH, M3U8Constants.M3U8_CODECS, M3U8Constants.M3U8_CLOSED_CAPTIONS},
-      new String[]{"1", "739000", "\"avc1.77.30, mp4a.40.2\"", "NONE"}),
-      createDto("https://hdvodsrforigin-f.akamaihd.net/i/vod/1gegen100/2010/05/1gegen100_20100517_200706_web_h264_16zu9_,lq1,mq1,hq1,.mp4.csmil/index_2_av.m3u8",
-      new String[]{M3U8Constants.M3U8_PROGRAM_ID, M3U8Constants.M3U8_BANDWIDTH, M3U8Constants.M3U8_CODECS, M3U8Constants.M3U8_CLOSED_CAPTIONS},
-      new String[]{"1", "1395000", "\"avc1.77.30, mp4a.40.2\"", "NONE"}),
-      createDto("https://hdvodsrforigin-f.akamaihd.net/i/vod/1gegen100/2010/05/1gegen100_20100517_200706_web_h264_16zu9_,lq1,mq1,hq1,.mp4.csmil/index_0_a.m3u8",
-      new String[]{M3U8Constants.M3U8_PROGRAM_ID, M3U8Constants.M3U8_BANDWIDTH, M3U8Constants.M3U8_CODECS, M3U8Constants.M3U8_CLOSED_CAPTIONS},
-      new String[]{"1", "23000", "\"mp4a.40.2\"", "NONE"}),
+        createDto(
+            "https://hdvodsrforigin-f.akamaihd.net/i/vod/1gegen100/2010/05/1gegen100_20100517_200706_web_h264_16zu9_,lq1,mq1,hq1,.mp4.csmil/index_0_av.m3u8",
+            new String[]{M3U8Constants.M3U8_PROGRAM_ID, M3U8Constants.M3U8_BANDWIDTH, M3U8Constants.M3U8_CODECS,
+                M3U8Constants.M3U8_CLOSED_CAPTIONS},
+            new String[]{"1", "118000", "\"avc1.66.30, mp4a.40.2\"", "NONE"}),
+        createDto(
+            "https://hdvodsrforigin-f.akamaihd.net/i/vod/1gegen100/2010/05/1gegen100_20100517_200706_web_h264_16zu9_,lq1,mq1,hq1,.mp4.csmil/index_1_av.m3u8",
+            new String[]{M3U8Constants.M3U8_PROGRAM_ID, M3U8Constants.M3U8_BANDWIDTH, M3U8Constants.M3U8_CODECS,
+                M3U8Constants.M3U8_CLOSED_CAPTIONS},
+            new String[]{"1", "739000", "\"avc1.77.30, mp4a.40.2\"", "NONE"}),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "\"avc1.77.30, mp4a.40.2\"" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)